examples/Makefile

```diff
@@ -1,7 +1,22 @@
-all: attosoc.json blinky.json
+DEVICE ?= GW1N-9
+PN ?= GW1NR-LV9QN881C6/I5
+
+all: attosoc.fs blinky.fs
+
+%.fs: %.pnr.json
+	gowin_pack -d ${DEVICE} -o $@ $^
+
+%.pnr.json: %.json
+	nextpnr-gowin --json $^ --write $@ --device ${PN} --cst tec0117.cst
 
 attosoc.json: attosoc/attosoc.v attosoc/picorv32.v
 	yosys -p "synth_gowin -json $@" $^
 
 %.json: %.v
 	yosys -p "synth_gowin -json $@" $^
+
+prog:
+	openFPGALoader -b littleBee -r -f blinky.fs
+
+clean:
+	rm -f *.json *.fs
```

Comment:
I think it's good style to make these targets phony?

Reply:
Ok, I will make the change.